src/pymor/bindings/dunegdt.py

```diff
@@ -0,0 +1,189 @@
+                      rescale_colorbars=False, block=None, filename=None, columns=2):
+
+            def visualize_single(vec, vec_name, f_name):
+                discrete_function = DiscreteFunction(self.space, vec, vec_name)
+                discrete_function.visualize(f_name[:-4])
+
+            suffix = '.vtp' if self.space.dimDomain == 1 else '.vtu'
+            interactive = filename is None
+            to_clean_up = []
+
+            if isinstance(U, tuple):
+                # we presume to have several vectors which should be visualized side by side
+                assert all([isinstance(u, ListVectorArray) for u in U])
+                assert all([len(u) == 1 for u in U])
+                assert all([all([isinstance(v, DuneXTVector) for v in u._list]) for u in U])
+                if legend is not None and len(legend) == len(U):
+                    names = legend
+                elif title is not None and len(title) == len(U):
+                    names = title
+                else:
+                    names = list()
+                    for ii in range(len(U)):
+                        names.append('STATE_{}'.format(ii))
+                data = OrderedDict()
+                if interactive:
+                    for ii in range(len(U)):
+                        _, filename = mkstemp(suffix='_{}{}'.format(ii, suffix))
+                        data[filename] = U[ii]._list[0]
+                else:
+                    if (filename.endswith('.vtp') or filename.endswith('.vtu')):
+                        filename = filename[:-4]
+                    assert len(filename) > 0
+                    for ii in range(len(U)):
+                        data['{}_{}{}'.format(filename, ii, suffix)] = U[ii]._list[0]
+                for name, f_name in zip(names, data):
+                    visualize_single(data[f_name].impl, name, f_name)
+                if interactive:
+                    _, pvd_filename = mkstemp(suffix='.pvd')
+                    with open(pvd_filename, 'w') as pvd_file:
+                        pvd_file.write('<?xml version="1.0"?>\n')
+                        pvd_file.write('<VTKFile type="Collection" version="0.1">\n')
+                        pvd_file.write('  <Collection>\n')
+                        part = 0
+                        for f_name in data:
+                            pvd_file.write('    <DataSet timestep="0" group="" part="{}" file="{}"/>\n'.format(part,
+                                                                                                               f_name))
+                            part += 1
+                        pvd_file.write('  </Collection>\n')
+                        pvd_file.write('</VTKFile>\n')
+                    subprocess.call(['paraview', pvd_filename])
+                    for f_name in data:
+                        to_clean_up.append(f_name)
+                    to_clean_up.append(pvd_filename)
+            else:
```

Comment:
Do you think it's possible to use a common code path for both branches here instead?

Reply:
... I would rather use `addFile` here, right? The case above is about putting several plots side-by-side, not about plotting a sequence. To merge these paths one would have to init the `vtp` file differently and then append the other files commonly, I guess. But not idea yet...

Reply:
Tbh I feel I don't understand the distinctions here very well. I expected something along the lines of flow in the built-in `PatchVisualizer`.